src/app/components/pages/send-skycoin/send-verify/send-verify.component.ts

```diff
@@ -0,0 +1,52 @@
+import { Component, EventEmitter, Input, Output, ViewChild } from '@angular/core';
+import { MatSnackBar, MatSnackBarConfig } from '@angular/material';
+
+import { PreviewTransaction } from '../../../../app.datatypes';
+import { WalletService } from '../../../../services/wallet.service';
+import { ButtonComponent } from '../../../layout/button/button.component';
+
+@Component({
+  selector: 'app-send-verify',
+  templateUrl: './send-verify.component.html',
+  styleUrls: ['./send-verify.component.scss'],
+})
+export class SendVerifyComponent {
+  @ViewChild('sendButton') sendButton: ButtonComponent;
+  @ViewChild('backButton') backButton: ButtonComponent;
+  @Input() transaction: PreviewTransaction;
+  @Output() onBack = new EventEmitter<boolean>();
+
+  constructor(
+    private walletService: WalletService,
+    private snackbar: MatSnackBar
+  ) {}
+
+  send() {
+    this.snackbar.dismiss();
+    this.sendButton.resetState();
+    this.sendButton.setLoading();
+
```

Comment:
`backButton` should be disabled here, to prevent the user from returning while the transaction is being sent.

Reply:
Fixed.